src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -313,10 +321,60 @@ export function InternalDashboardTopNav({
       });
     }
     if (showWriteControls && managed) {
-      allBadges.push(getManagedContentBadge(dashboardManagedBadge.getTooltip()));
+      const badgeProps = {
+        ...getManagedContentBadge(dashboardManagedBadge.getAriaLabel(), false),
+        onClick: () => setIsPopoverOpen(!isPopoverOpen),
+        onClickAriaLabel: dashboardManagedBadge.getAriaLabel(),
+        iconOnClick: () => setIsPopoverOpen(!isPopoverOpen),
+        iconOnClickAriaLabel: dashboardManagedBadge.getAriaLabel(),
+      } as TopNavMenuBadgeProps;
+      allBadges.push({
+        renderCustomBadge: ({ badgeText }) => {
+          const badgeButton = <EuiBadge {...badgeProps}>{badgeText}</EuiBadge>;
+          return (
+            <EuiPopover
+              button={badgeButton}
+              isOpen={isPopoverOpen}
+              closePopover={() => setIsPopoverOpen(false)}
+              panelStyle={{ maxWidth: 250 }}
+            >
+              <FormattedMessage
+                id="dashboard.managedContentPopoverButton"
+                defaultMessage="Elastic manages this dashboard.
+                {Duplicate} it to make changes."
```

Comment:
**nit** The new line isn't actually being applied to the formatting, so we should remove it

```suggestion
                defaultMessage="Elastic manages this dashboard. {Duplicate} it to make changes."
```